packages/kbn-reporting/public/share/share_context_menu/screen_capture_panel_content.test.tsx

```diff
@@ -30,7 +30,7 @@ const getJobParamsDefault = () => ({
 
 const theme = themeServiceMock.createSetupContract();
 
-test('ScreenCapturePanelContent renders the default view properly', () => {
+xtest('ScreenCapturePanelContent properly renders a view with "canvas" layout option', () => {
```

Comment:
looks like we've got a typo here :)

Reply:
Good catch - will be doing test cleanup in a subsequent PR, just reverted this to main and should be passing!